src/controllers/chatController.ts

```diff
@@ -0,0 +1,16 @@
+import Chat from '../models/chat/chat';
+import { getChats } from '../services/chats';
+import { State } from '../utils/enums';
+
+export default class ChatContoler {
+  getAll() : Array<Chat> {
+    let chats = getChats();
+    return chats;
```

Comment:
Можно сразу возвращать результат, не записывая в новую переменную 
```suggestion
    return getChats();
```